subsys/bluetooth/controller/Kconfig

```diff
@@ -7,6 +7,48 @@
 
 comment "BLE Controller support"
 
+# The following symbols are enabled depending if the controller actually
+# supports the respective features.
+
```

Comment:
Maybe we should add `BT_CTRL_SLAVE_SUPPORT` and `BT_CTRL_MASTER_SUPPORT` as well?This will allow us to add ifdefs for master and slave specific code

Reply:
Yes, I think it makes sense. But I'd rather do this as a separate PR, after the current one gets merged. Is this acceptable?

Reply:
What's wrong with using the already existing BT_CENTRAL/BT_PERIPHERAL?

Reply:
I see your point. This CENTRAL/PERIPHERAL may be sufficient. CENTRAL/PERIPHERAL is host terminology, while MASTER/SLAVE is controller terminology

Reply:
CENTRAL/PERIPHERAL are part of the common Kconfig variables for host and controller.
The question is whether we should add _SUPPORT kconfig options for them. And for those it would make sense to use controller MASTER/SLAVE terminology. So that it is possible to add controller with only SLAVE support for example.

Reply:
I do not see the use case for having a separate MASTER/SLAVE configuration flag for the controller. If the controller supports only SLAVE, then compiling the host with CENTRAL support would not make a lot of sense.

Reply:
Is this discussion relevant for Vega support?

Reply:
It's relevant to the _SUPPORT change that was done as part of this PR.
But alexandru suggested to do this change separately and we agreed.. So we can end this discussion here.